docs/modules/reference/pages/how-to-build-with-docker.adoc

```diff
@@ -1,29 +1,31 @@
+
 = Build with Docker
 
+This section describes how to build {page-component-title} with Docker.
+
 == Simple build
 
-Helm can be compiled from source with a Docker build image.
+{page-component-title} can be compiled from source with a Docker build image:
 
+. Check out the repository, compile the code, and install the plugin:
++
 [source, console]
-----
 docker build -t myopennmshelmimage .
```

Comment:
```suggestion
docker build -t myopennmshelmimage 
```
not sure if the period was required. It looked odd.

Reply:
I think it might be required? I'll defer to others who are more familiar with console commands, though.